foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/helper/ConversionManager.java

```diff
@@ -44,6 +48,31 @@
  */
 public class ConversionManager extends CoreConversionManager implements Serializable, Cloneable {
     protected Map defaultNullValues;
+    private static final ZoneId defaultZoneOffset;
+    
+    static {
+        ZoneId tzoneid = null;
+        String tzone = PrivilegedAccessHelper.getSystemProperty("org.eclipse.persistence.conversion.useTimeZone");
+        if (tzone != null) {
+            
+            try {
+                tzoneid = java.time.ZoneId.of(tzone);
+            } catch (Throwable t) {
+                // If an invalid time zone id is supplied, then fall back to checking for checking for using
+                // either UTC or the system's default time zone.
+            }
+        } 
+        if (tzoneid == null) {
+            String propVal = PrivilegedAccessHelper.getSystemProperty("org.eclipse.persistence.conversion.useDefaultTimeZoneForJavaTime", "false");
+            if (Boolean.parseBoolean(propVal)) {
+                tzoneid = java.time.ZoneId.systemDefault();
+            } else {
+                tzoneid = ZoneOffset.UTC;
+            }
+        }
+        
+        defaultZoneOffset = tzoneid;
+    }
```

Comment:
Pehaps it may be better design to use use the lazy initialization pattern similar to how `org.eclipse.persistence.internal.helper.ConversionManager.defaultManager` is initialized?

Reply:
The value wouldn't be initialized until the ConversionManager type is first loaded.  This is a quick operation that only occurs once -- when the class is loaded.

Reply:
I am suggesting that we should not use static blocks. I think a cleaner implementation would be using lazy initialization of a singleton.

Reply:
I agree with @dazey3 here

Reply:
I accept the suggestion to change it.

Reply:
Addressed, please re-review.